pkg/archive/archive.go

```diff
@@ -124,15 +125,66 @@ func IsArchivePath(path string) bool {
 	return err == nil
 }
 
+const (
+	ZstdMagicSkippableStart = 0x184D2A50
```

Comment:
Linter is failing on these consts and vars because they're missing a GoDoc. It looks like we're only using it internally in this package (at least currently), so I suggest to un-export them;


```
pkg/archive/archive.go:129:2: exported const ZstdMagicSkippableStart should have comment (or a comment on this block) or be unexported (golint)
	ZstdMagicSkippableStart = 0x184D2A50
	^
pkg/archive/archive.go:134:2: exported var `Bzip2Magic` should have comment or be unexported (golint)
	Bzip2Magic = []byte{0x42, 0x5A, 0x68}
	^
```

Reply:
Done.